pkg/landscape/app/group-view.hoon

```diff
@@ -436,13 +443,14 @@
       ?.  ?=(%initial-group -.update)  jn-core
       ?.  =(group.update rid)          jn-core
       |^  ^+  jn-core
+      =/  =request:view  (~(got by joining) rid)
       =/  feed  feed-rid
       =.  jn-core  (cleanup %done)
       =/  hidden  hidden:(need (scry-group:grp rid))
       =?  jn-core  ?&(!hidden ?=(^ feed))
         %-  emit
         (pull-gra:pass (need feed))
-      =?  jn-core  !hidden
+      =?  jn-core  |(hidden autojoin.request)
```

Comment:
Previously, we would pull if it wasn't hidden. Now, we pull when it's hidden or we want to autojoin. Feels like this should be `&(!hidden autojoin.request)` maybe?

Reply:
Hm? Either it's hidden (i.e is an unmanaged group) or the user has manually selected that they wish to join all graphs

Reply:
Ok, but this is clearly different behavior from the previous version, where we only emitted these cards if it was _not_ hidden. Was that just a faulty implementation? Just making sure that checks out with the indented behavior here (which I admittedly do not fully understand, in either the old or new context).